techniques/html/H43.html

```diff
@@ -51,46 +51,44 @@
 
 <div class="default-grid with-gap leftcol">
 <aside class="box nav-hack sidebar standalone-resource__sidebar">
-    <header class="box-h">Page Contents</header>
+    <header class="box-h">このページの内容</header>
     <div class="box-i">
       <nav aria-label="page contents" class="navtoc">
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#resources">Related Resources</a></li>
-<li><a href="#related">Related Techniques</a></li>
-<li><a href="#tests">Tests</a></li>
-<li><a href="#test-rules">Test Rules</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#resources">関連リソース</a></li>
+<li><a href="#related">関連テクニック</a></li>
+<li><a href="#tests">検証</a></li>
+<li><a href="#test-rules">テストルール</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
       
-<h1><span>Technique H43:</span>Using <code class="language-html">id</code> and <code class="language-html">headers</code> attributes to associate data cells with header cells in data tables</h1>
+<h1><span>テクニック H43:</span>データテーブルのデータセルを見出しセルと関連付けるために、<code class="language-html">id</code> 属性及び <code class="language-html">headers</code> 属性を使用する</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
   
   <p>
-    This technique relates to
+    このテクニックは
     
 <a href="https://w3c.github.io/wcag/understanding/info-and-relationships">1.3.1: Info and Relationships</a>
-(<strong>Sufficient</strong>
-  when used for making information and relationships conveyed through presentation programmatically
-                     determinable).</p>
+(提示を通じて伝えられる情報及び関係性をプログラムによる決定が可能にするために使用される場合に<strong>十分なテクニック</strong>) に関連する。</p>
```

Comment:
programmatically determinable を「プログラムによる解釈が可能」としました。
```suggestion
(提示を通じて伝えられる情報及び関係性をプログラムによる解釈が可能にするために使用される場合に<strong>十分なテクニック</strong>) に関連する。</p>
```